packages/admin/src/pages/super-admin-dashboard/user/[id]/schemes/[schemeId]/change-owner.test.tsx

```diff
@@ -128,7 +128,7 @@ describe('Super admin - Change owner page', () => {
         schemeId: 'testSchemeId',
       },
       query: {
-        oldEmailAddress: 'test%40gmail.com',
+        oldEmailAddress: 'oldemail@gov.uk',
```

Comment:
Ideally also want to test it can decode it
```suggestion
        oldEmailAddress: 'oldemail%40gov.uk',
```